### PR TITLE
Add EMR 6.3 documentation

### PR DESCRIPTION
**Pull request by viadea**

### CODE REVIEW

File: docs/get-started/getting-started-aws-emr.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ This is a getting started guide for the RAPIDS Accelerator for Apache Spark on A
 of this guide, the user will be able to run a sample Apache Spark application that runs on NVIDIA
 GPUs on AWS EMR.
 
-The current EMR 6.2.0 release supports Spark version 3.0.1 and RAPIDS Accelerator version 0.2.0. For
+The current EMR 6.3.0 release supports Spark version 3.1.1-amzn-0 and RAPIDS Accelerator version 0.4.1.
```
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:13]: The "amzn-0" is implementation detail, users will know it as Spark 3.1.1.
```suggestion
The current EMR 6.3.0 release supports Spark version 3.1.1 and RAPIDS Accelerator version 0.4.1.
```

File: docs/get-started/getting-started-aws-emr.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ This is a getting started guide for the RAPIDS Accelerator for Apache Spark on A
...
+EMR 6.2.0 release supports Spark version 3.0.1 and RAPIDS Accelerator version 0.2.0. For
```
[the remark this answers is not in the document]
Comment (viadea) [reply docs/get-started/getting-started-aws-emr.md]: Sure, I just added a version matrix for EMR for :EMR , Spark, Rapids accelerator jar , cuDF jar and xgboost4j-spark jar.
How about now?

File: docs/get-started/getting-started-aws-emr.md [hunk 3]
```diff
@@ -10,8 +10,14 @@ This is a getting started guide for the RAPIDS Accelerator for Apache Spark on A
 of this guide, the user will be able to run a sample Apache Spark application that runs on NVIDIA
 GPUs on AWS EMR.
 
-The current EMR 6.2.0 release supports Spark version 3.0.1 and RAPIDS Accelerator version 0.2.0. For
-more details of supported applications, please see the [EMR release
+Different versions of EMR ship with different versions of Spark, Rapids accelerator jar , cuDF jar and xgboost4j-spark jar:
```
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:13]:
```suggestion
Different versions of EMR ship with different versions of Spark, RAPIDS Accelerator, cuDF and xgboost4j-spark:
```

File: docs/get-started/getting-started-aws-emr.md [hunk 4]
```diff
@@ -10,8 +10,14 @@ This is a getting started guide for the RAPIDS Accelerator for Apache Spark on A
 of this guide, the user will be able to run a sample Apache Spark application that runs on NVIDIA
 GPUs on AWS EMR.
 
-The current EMR 6.2.0 release supports Spark version 3.0.1 and RAPIDS Accelerator version 0.2.0. For
-more details of supported applications, please see the [EMR release
+Different versions of EMR ship with different versions of Spark, RAPIDS Accelerator, cuDF and xgboost4j-spark:
+
+| EMR | Spark | Rapids Accelerator jar | cuDF jar | xgboost4j-spark jar
```
Comment (jlowe) [inline docs/get-started/getting-started-aws-emr.md:15]: oops, I missed this one in the last review.  RAPIDS should always be caps
```suggestion
| EMR | Spark | RAPIDS Accelerator jar | cuDF jar | xgboost4j-spark jar
```

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

**Pull request merged**